doc/support_matrix.yml

```
@@ -309,6 +309,21 @@ chips:
           - removing items not supported
       wifi: not_available
 
+  stm32wba55cg:
+    name: STM32WBA55CG
+    support:
+      gpio: supported
+      debug_output: supported
+      hwrng: supported
+      i2c_controller: supported
+      spi_main: supported
+      logging: supported
+      storage:
+        status: supported_with_caveats
```

```suggestion
        status: not_currently_supported
```
We would need to at least add something in [`ariel-os-storage/build.rs`](https://github.com/ariel-os/ariel-os/blob/fbc504587b18fa2e1eb962398e24a4bf71ba315e/src/ariel-os-storage/build.rs) for this be supported (adding support for anything is out of scope of this PR).